app/views/appointments/calendar.html.erb

```diff
@@ -0,0 +1,71 @@
+<% content_for(:head) do %>
+  <%= stylesheet_link_tag "fullcalendar/dist/fullcalendar" %>
+  <%= stylesheet_link_tag "fullcalendar-scheduler/dist/scheduler" %>
+<% end %>
+
+<div class="page-header">
+  <ol class="breadcrumb">
+    <li><a href="<%= root_path %>">Appointment planner</a></li>
+    <li><a href="<%= appointments_url %>">Appointments</a></li>
+    <li class="active">Appointments calendar</li>
+  </ol>
+  <h1>Appointments calendar</h1>
+</div>
+
+<div class="row">
+  <div class="col-md-10">
+    <div id="calendar"></div>
+  </div>
+  <div class="col-md-2 filters">
+    <h3>Options</h3>
+
+    <h4>Guiders</h4>
+    <% @booking_location.guiders.each do |guider| %>
+      <p>
+        <label for="guider_<%= guider.id %>"><input type="checkbox" id="guider_<%= guider.id %>" name="guider_<%= guider.id %>" value="1" checked> <%= guider.name %></label>
+      </p>
+    <% end %>
+    <hr>
+    <h4>Locations</h4>
+    <% @locations_all.each do |location| %>
+      <p>
+        <label for="location_<%= location[1] %>"><input type="checkbox" id="location_<%= location[1] %>" name="location_<%= location[1] %>" value="1" checked> <%= location[0] %></label>
+      </p>
+    <% end %>
+  </div>
+</div>
+
+<script>
+window.PWPLAN = window.PWPLAN || {};
+
+$(function() {
```

Comment:
Seeing this stuff now (and how much nicer it is having the js separated for the rest of it), it probably makes sense to convert this to a feed/endpoint of sorts. I guess we can chat through this with @benlovell too